roles/ocp/tasks/creds.yml

```diff
@@ -31,4 +31,13 @@
         src: cloud-creds.j2
         dest: "{{ lookup('env', 'HOME') }}/.{{ creds_path }}/{{ creds_file }}"
         mode: 0600
-  when: not cloud_creds.stat.exists
+  when: not cloud_creds.stat.exists or item.cloud.platform == 'gcp'
```

Comment:
I would say that:
`when: item.cloud.platform == gcp`
will be good.
Since currently only gcp has a requirement to create a separate file.

Also, no need to check for cloud_creds file here and below.
Instead I suggest to check for `cloud_creds_path` (only the path where the file should reside) and file will be created or update in any case.